graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/UnsupportedRemoteReindexingMigrationAdapterES7.java

```diff
@@ -30,6 +31,11 @@ public class UnsupportedRemoteReindexingMigrationAdapterES7 implements RemoteRei
 
     public static final String UNSUPPORTED_MESSAGE = "This operation should never be called. We remote-reindex into the DataNode that contains OpenSearch. This adapter only exists for API completeness";
 
+    @Override
+    public boolean isMigrationRunning(IndexSet indexSet) {
+        throw new UnsupportedOperationException(UNSUPPORTED_MESSAGE);
```

Comment:
We should not throw an exception here as it breaks retention when running against es.
Actually, this must also be fixed for `getLatestMigrationId`

Reply:
Oh yes, you are right! Will fix that ASAP.